lms/envs/aws.py

```diff
@@ -864,3 +864,12 @@
 APP_UPGRADE_CACHE_TIMEOUT = ENV_TOKENS.get('APP_UPGRADE_CACHE_TIMEOUT', APP_UPGRADE_CACHE_TIMEOUT)
 
 AFFILIATE_COOKIE_NAME = ENV_TOKENS.get('AFFILIATE_COOKIE_NAME', AFFILIATE_COOKIE_NAME)
+
+################################ Edraak Apps ################################
+
+if FEATURES.get('ENABLE_EDRAAK_FORUS'):
```

Comment:
Please rebase on `master` and move all the `/envs/` (the four of them) to the other `Edraak Apps` sections.